samples/openapi3/client/petstore/dart-dio/oneof_polymorphism_and_inheritance/lib/src/api.dart

```diff
@@ -27,8 +27,8 @@ class Openapi {
         this.dio = dio ??
             Dio(BaseOptions(
```

Comment:
Now `dio` 5.0 will not imply the default content type, please ensure it's valid in requests.

Reply:
We have always manually set the content-type if it is specified in the spec. See https://github.com/OpenAPITools/openapi-generator/blob/master/samples/openapi3/client/petstore/dart-dio/petstore_client_lib_fake/lib/src/api/fake_api.dart#L147

Reply:
I think thats enough for now.